course_catalog/admin.py

```diff
@@ -149,21 +149,38 @@ class PodcastEpisodeAdmin(admin.ModelAdmin):
 class EnrollmentAdmin(admin.ModelAdmin):
     """Enrollment Admin"""
 
-    def run_slug(self, obj):
-        """run slug as string"""
-        return f"{obj.run.slug}"
-
-    run_slug.admin_order_field = "run__slug"
+    def run_id(self, obj):
+        """run_id as string"""
+        if obj.run:
+            return f"{obj.run.run_id}"
+        else:
+            return ""
+
+    def course_id(self, obj):
+        """course_id as string"""
+        if obj.course:
+            return f"{obj.course.course_id}"
+        else:
+            return ""
 
     model = Enrollment
     search_fields = (
+        "course__course_id",
+        "course__title",
+        "enrollments_table_run_id",
```

Comment:
What changes are you making the admin here?

Reply:
I'm making changes to the Enrollments admin page (currently no data).

There aren't matches for runs for all enrollments. 
This updates the admin page to display just the course match for courses where there is a match on the course id but not the run id. It also shows the run id from the enrollments table